manifests/process_exporter.pp

```diff
@@ -107,12 +111,19 @@
     default => undef,
   }
 
+  if empty($watched_processes) {
```

Comment:
since the default is undef, we don't need to check for empty string, we only need to validate if it's defined:
```suggestion
  if ($watched_processes) {
```

Reply:
puppet doesn't require parentheses after if 😜

Reply:
no, Puppet doesn't :) You can add them, but they don't bring any value I think.